lib/smart_answer_flows/simplified-expenses-checker.rb

```diff
@@ -285,77 +200,83 @@ def define
 
       #Q14 - people who live on business premises?
       value_question :people_live_on_premises?, parse: :to_i do
-        calculate :live_on_premises do |response|
-          response
+        next_node do |response|
+          calculator.hours_lived_on_business_premises = response
+          outcome :you_can_use_result
         end
+      end
 
-        calculate :simple_business_costs do
-          amount = case live_on_premises
-                   when 0 then 0
-                   when 1 then 4200
-                   when 2 then 6000
-                   else 7800
-                   end
+      outcome :you_cant_use_result
+      outcome :you_can_use_result do
+        precalculate :vehicle_is_green do
```

Comment:
yeah that can be done in another commit.. I only thought it best to avoid making changes in the erb templates due to undefined exceptions etc